src/test/java/seemedics/service/triage/engine/IntegrationTestTriageProtocolsSourceConfiguration.java

```diff
@@ -0,0 +1,26 @@
+package seemedics.service.triage.engine;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Created by igor-z on 29-Apr-17.
+ */
+@Configuration
```

Comment:
I really happy you are into it:)
Now we can define integration tests that use multiple files that are defined in our project!!!!
This is cool.
A few notes:

1. You can use the DefaultTraigeProtocolsSource instead IntegrationTestTriageProtocolsSource (the only diff is the configuration - the path)
2. When you override the DefaultTraigeProtocolsSource you actually dont test it (so it is always better run the same code for tests as for production and change only config) - in this case it is possible.
3. For now the main app failed start since DefaultTraigeProtocolsSource is not defined.